src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -51,10 +51,12 @@ const ArticleTimestamp = ({ firstPublished, lastPublished, popOut }) => {
 
   const Wrapper = popOut ? PopOutGridItemMedium : GridItemConstrainedMedium;
 
+  const timestampDifference = (lastPublished - firstPublished) / 1000 / 60;
```

Comment:
Can we rename the variable such that it is clear that the difference is in minutes?

Reply:
Alternatively, you could use Moment.js e.g https://momentjs.com/docs/#/durations/diffing/

Reply:
Done :)